### ISSUE TITLE
Radio doesnt work after enabling replay gain option

### ISSUE THREAD

**Issue by nunotfc** [AzuraCast/AzuraCast#5093]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

ac41b6d (2022-02-12 8:11) • Docker • PHP 8.1

### What happened?

hello, after enabling replay gain option my radio plays azuracast is live! and keeps stuck.
i send the liquidsoap log
https://www.renovagames.com/log_azura.txt

**Comment by SC2Mitch** [comment]
Can you provide us your liquidsoap config?

Also how do you enable this feature? I'll give it a test in a moment.

**Comment by nunotfc** [comment]
i just wentt to settings, autodj and enable use replay gain metadata
my liquidsoap config is jut one line different than the original
https://www.renovagames.com/config_azura.txt

**Comment by SlvrEagle23** [comment]
@nunotfc Is there a chance you can host that log somewhere else, like a Pastebin? My antivirus is throwing a fit about that URL for some reason.

**Comment by nunotfc** [comment]
hello, just to inform that i continue with this ishue, ho mather updates i made. im at 
57e2856 (2022-02-23 1:00) • Docker • PHP 8.1

**Comment by nunotfc** [comment]
hello! just to inform thath this bug persists.

**Comment by SlvrEagle23** [comment]
@nunotfc Does it persist after the very latest update (which came out just after that one)?

**Comment by nunotfc** [comment]
hello, just to inform that updating to ls 2.0.4 still doesnt fix the bug. is there any log i can provide? or can i give the access to my station? if i can do something, i will make. i dont see anyone with this problem... is it possible that ls broke when some track doesnt have gain information? i have some tracks with this information, and some withouth.

**Comment by nunotfc** [comment]
what i get is like this:
2022/03/18 13:43:25 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621805.89)
2022/03/18 13:43:25 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621805.89)
2022/03/18 13:43:25 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621805.89)
and this:
2022/03/18 13:44:43 [request:2] Time limit exceeded by 67.00 secs!

**Comment by Vaalyn** [comment]
Could you post a bit more of the log?

Without seeing the full chain of logs from LS receiving the response of what to play next we can't really see what's going on.

**Comment by nunotfc** [comment]
the log is stuck, after last line.
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.81)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.84)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.84)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.84)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.84)
2022/03/18 13:43:38 [lang:3] range_start <= current_time and current_time <= range_end = false (1647621818.84)
2022/03/18 13:43:42 [request:2] Time limit exceeded by 5.00 secs!
2022/03/18 13:43:43 [lang:3] API nextsong - Response: annotate:title="PASSAGEM - RENOVA RADIO 2020 - 03 - MASTER",artist="Renova Radio Promo",duration="3.00",song_id="b93b85da3bc318d5926ae184e0367c92",media_id="568",liq_amplify="0.00dB",jingle_mode="true":/var/azuracast/stations/renovaradio/media/promos/PASSAGEM - RENOVA RADIO 2020 - 03 - MASTER.mp3
[mp3 @ 0x7f59b4018e80] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
2022/03/18 13:44:43 [request:2] Time limit exceeded by 67.00 secs!
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b403dbc0] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b0028e40] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b0027e80] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b0028e40] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f59b0028e40] Estimating duration from bitrate, this may be inaccurate
2022/03/18 13:45:38 [next_song:3] Prepared "/var/azuracast/stations/renovaradio/media/promos/PASSAGEM - RENOVA RADIO 2020 - 03 - MASTER.mp3" (RID 20).
2022/03/18 13:45:38 [lang:3] AutoDJ is ready!
2022/03/18 13:45:38 [lang:3] API nextsong - Response: annotate:title="One Time (Official Music Video)",artist="MK xyz",duration="157.00",song_id="3c13c39b927f3610af1c140e5f6358a9",media_id="832",liq_amplify="0.00dB",playlist_id="6":/var/azuracast/stations/renovaradio/media/gerais/MK xyz - One Time (Official Music Video).mp3

**Comment by nunotfc** [comment]
in radio, i keep listening a noize like rain or fm sound.

**Comment by Vaalyn** [comment]
Can't see anything unusual in those logs 🤔 

> searching a listle bit, i get this error, berfore starting the numeric lines.
2022/03/18 13:56:30 [error_jingle:3] /usr/local/share/icecast/web/error.mp3 is static, resolving once for all...

This is not an error the `error_jingle:3` means this is a log entry generated for the following fallback:
```
radio = fallback(id="safe_fallback", track_sensitive = false, [radio, single(id="error_jingle", "/usr/local/share/icecast/web/error.mp3")])
```

As you can see in there the `single()` method has the `id` named `error_jingle`.

**Comment by nunotfc** [comment]
the full log is here:
https://www.renovagames.com/log.txt
sorry, pastebin doesnt allow me, because of the limit.

**Comment by Vaalyn** [comment]
Apart from the amount of date range entries there is only this line that looks a bit weird to me:
```
2022/03/18 13:56:31 [buffer.consumer_0:3] Source failed (no more tracks) stopping output...
```

How does the general resource usage look like of the installation when you take a look at the stats on the Administration page?

**Comment by nunotfc** [comment]
CPU
 
Total
36
    Steal : 0.00%
    Espera : 0.00%
    Uso : 36.35%
 
Cpu0
4
St: 0.00%
Wa: 0.00%
Us: 3.96%
Cpu1
2
St: 0.00%
Wa: 0.00%
Us: 2.04%
Cpu2
100
St: 0.00%
Wa: 0.00%
Us: 100.00%
Cpu3
6
St: 0.00%
Wa: 0.00%
Us: 6.00%
Cpu4
100
St: 0.00%
Wa: 0.00%
Us: 100.00%
Cpu5
4
St: 0.00%
Wa: 0.00%
Us: 4.17%
Média de Consumo
1-Min
2.64
5-Min
1.62
15-Min
0.99
Memória
 
Total de RAM : 16.4 GB
1.27686e+10
3.40845e+09
    Usado : 12.8 GB
    Em Cache : 3.4 GB
Espaço em Disco
Espaço Total em Disco : 264.2 GB
9.05851e+10
    Usado : 90.6 GB
Interfaces de Rede
ETH0
LO
Recebido
Speed
Packets
Errs
Drop
Fifo
Frame
Compressed
Multicast
330.0 B/s
5
0
0
0
0
0
0
Transmitido
Speed
Packets
Errs
Drop
Fifo
Frame
Carrier
Compressed
16.3 KB/s
5
0
0
0
0
0

**Comment by nunotfc** [comment]
hello,
i changed server, updated to latest commit and the bug still present.

**Comment by nunotfc** [comment]
hello! just to inform that problem is solved. but everytime i restart server i have to wait 1,2 minutes with the error sound to start work with replay gain enabled. if not enabled, the radio start playing inmediately. there is something i can do to solve it?

**Comment by nunotfc** [comment]
hello, just to inform. after some days working properly the bug started to ocur again. restarting doesnt work, just error sound like 1 2 minutes and the statick feedback sound. only disabling replaygain worked. none of the optimizations work to fix that.

**Comment by toots** [comment]
Hi! Liquidsoap dev here. We are aware of a rare case of application crash with the `%mp3` (lame) encoder that can be caused by invalid audio data (`nan` audio samples). This crash should be fixed with the upcoming `2.0.4` release. See: https://github.com/savonet/ocaml-lame/issues/4
